packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendarCell.tsx

```diff
@@ -1,9 +1,67 @@
-import { compose } from '@fluentui/react-bindings';
-import { Button, ButtonProps, ButtonStylesProps } from '../Button/Button';
+import {
+  Accessibility,
+  datepickerCalendarCellBehavior,
+  DatepickerCalendarCellBehaviorProps,
+} from '@fluentui/accessibility';
+import {
+  ComponentWithAs,
+  getElementType,
+  useAccessibility,
+  useStyles,
+  useFluentContext,
+  useTelemetry,
+  useUnhandledProps,
+  compose,
+} from '@fluentui/react-bindings';
+import * as _ from 'lodash';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+import { ComponentEventHandler, ComponentKeyboardEventHandler } from '../../types';
+import { commonPropTypes, UIComponentProps } from '../../utils';
 
-export type DatepickerCalendarCellProps = {};
+export interface DatepickerCalendarCellProps extends UIComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   */
+  accessibility?: Accessibility<DatepickerCalendarCellBehaviorProps>;
 
-export type DatepickerCalendarCellStylesProps = ButtonStylesProps;
+  /** Cell's primary content. */
+  label?: string;
+
+  /**
+   * Called on click.
+   *
+   * @param event - React's original SyntheticEvent.
+   * @param data - All props.
+   */
+  onClick?: ComponentEventHandler<DatepickerCalendarCellProps>;
+
+  /**
+   * Called on focus.
+   *
+   * @param event - React's original SyntheticEvent.
+   * @param data - All props.
+   */
+  onFocus?: ComponentEventHandler<DatepickerCalendarCellProps>;
+
+  /** A cell can show that it cannot be interacted with. */
+  disabled?: boolean;
+
+  /** A cell can show that it is currently selected or not. */
+  selected?: boolean;
```

Comment:
Do we use `selected` or `active` for this?

Reply:
We use `active` in Carousel, Toolbar, Menu, Accordion and `selected` in Tree, List, Table, Dropdown.
I'd say that semantically `selected` is better here

Reply:
Has sense